### DISCUSSION TITLE
Some questions

### DISCUSSION THREAD

**Discussion by DJHammers** [lavalink-devs/Lavalink#812] [Q&A] [answered]
Is idel players use lot of lavalink cpu ?
is there any way to reduce high cpu usage

**Comment by devoxin** [comment, marked as answer]
Idle players should use negligible amounts of CPU. If you're using a lot of CPU you could either upgrade the server/hardware Lavalink is running on (in the case of less powerful servers), or lock out `volume`/`filter` operations. Transcoding audio is a taxing operation on the CPU. Sources such as YouTube and SoundCloud are already encoded as opus, so these benefit from needing no transcoding. As soon as you alter the volume or add in any audio filters, the audio needs to be transcoded which uses considerably more CPU.
